webapp/src/ts/services/check-date.service.ts

```diff
@@ -0,0 +1,61 @@
+import { HttpClient } from '@angular/common/http';
+import { Injectable } from '@angular/core';
+
+import { TelemetryService } from './telemetry.service';
+import { ModalService } from '@mm-modals/mm-modal/mm-modal';
+import { CheckDateComponent } from '@mm-modals/check-date/check-date.component';
+
+const A_DATE_IN_THE_PAST = 1454424982000;
```

Comment:
I feel like we can update this, no? :D It's been more than 4 years. https://github.com/medic/cht-core/commit/82fafe9ce4e10348da695389f17bb0a371faaea5

Reply:
Yes good idea, I'll set the timestamp to the date of today.